src/workerd/io/worker-interface.capnp

```diff
@@ -146,6 +146,7 @@ struct QueueMessage @0x944adb18c0352295 {
   timestampNs @1 :Int64;
```

Comment:
Need some guidance regarding the id @0x944adb18c0352295

Reply:
What guidance do you need? We shouldn't need to do anything with the ID, all we're doing is adding a field to the struct.